sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -35,19 +31,132 @@ export class ContainerRegistryUserCredential {
     }
 
 // @public
-export type ContentProperties = ChangeableAttributes;
+export class ContainerRepositoryClient {
+    constructor(endpointUrl: string, repository: string, credential: TokenCredential | ContainerRegistryUserCredential, options?: ContainerRegistryClientOptions);
+    delete(options?: DeleteOptions): Promise<DeleteRepositoryResult>;
+    deleteRegistryArtifact(digest: string, options?: DeleteRegistryArtifactOptions): Promise<RestResponse>;
+    deleteTag(tag: string, options?: DeleteTagOptions): Promise<RestResponse>;
+    endpoint: string;
+    getProperties(options?: GetPropertiesOptions): Promise<RepositoryProperties>;
+    getRegistryArtifactProperties(tagOrDigest: string, options?: GetRegistryArtifactPropertiesOptions): Promise<RegistryArtifactProperties>;
+    getTagProperties(tag: string, options?: GetTagPropertiesOptions): Promise<TagProperties>;
+    listRegistryArtifacts(options?: ListRegistryArtifactsOptions): PagedAsyncIterableIterator<RegistryArtifactProperties>;
+    listTags(options?: ListTagsOptions): PagedAsyncIterableIterator<TagProperties>;
+    registry: string;
+    repository: string;
+    setManifestProperties(digest: string, value: ContentProperties, options?: SetManifestPropertiesOptions): Promise<RestResponse>;
+    setPermissions(value?: ContentProperties, options?: SetPermissionsOptions): Promise<RestResponse>;
```

Comment:
You can call setPermissions without any values?

Reply:
No, not intended. I don't know why I have a {} as default value in the code. Fixing